spyvm/squeakimage.py

```diff
@@ -259,8 +260,8 @@ class SqueakImage(object):
     exec "\n".join(code)
 
     def __init__(self, reader):
-        space = reader.space
-        self.special_objects = reader.special_w_objects
+        space = self.space = reader.space
+        self.special_objects = space.wrap_list(reader.special_w_objects)
```

Comment:
As per the naming conventions this should be called special_objects_w (wrapped list) then.